lmdeploy/model.py

```diff
@@ -707,29 +707,75 @@ def messages2prompt(self, messages, sequence_start=True):
 @MODELS.register_module(name='falcon')
 class Falcon(BaseModel):
 
-    def __init__(self):
-        super().__init__()
+    def __init__(self, **kwargs):
```

Comment:
Seems falcon is not a chat model. If it can only do text completion, may just inherit from BaseModel and do nothing.